webblog/webblog/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for webblog project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/howto/deployment/wsgi/
+"""
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'webblog.settings')
```

Comment:
by default it should be pointing towards local settings `webblog.settings.local`

Reply:
implemented the local and also set the default too.